core/lexicon/en/workspace.inc.php

```diff
@@ -171,6 +171,7 @@
 $_lang['provider_confirm_remove'] = 'Are you sure you want to delete this Provider?';
 $_lang['provider_err_blank_response'] = 'MODX received a blank response from the provider. Please double check your service URL and make sure the provider is a valid provider.';
 $_lang['provider_err_connect'] = 'An error occurred while connecting to the Provider: [[+error]]';
+$_lang['provider_err_invalid_xml'] = 'The provider did not return valid XML. Make sure the service url is valid and that the package provider is currently available.';
```

Comment:
```suggestion
$_lang['provider_err_invalid_xml'] = 'The provider did not return valid XML. Make sure the service URL is valid and that the package provider is currently available.';
```

To make it more consistent with `provider_err_blank_response` on line 172.